plugin/path_config.go

```diff
@@ -67,6 +66,10 @@ func (b *backend) configFields() map[string]*framework.FieldSchema {
 		Default:     defaultPasswordLength,
 		Description: "The desired length of passwords that Vault generates.",
 	}
+	fields["formatter"] = &framework.FieldSchema{
+		Type:        framework.TypeString,
+		Description: "Text to insert the password into, ex. \"customPrefix{{PASSWORD}}customSuffix\".",
```

Comment:
Optional: You can avoid escaping the double quotes by using backticks.

Reply:
Ah right, will do.